src/components/NoteDialogComponents/NoteDialogNoteComponents/NoteDialogNoteContent.tsx

```diff
@@ -26,21 +28,32 @@ export const NoteDialogNoteContent: FC<NoteDialogNoteContentProps> = ({noteId, a
     dispatch(Actions.onNoteBlur());
   };
 
+  const isImage = useImageChecker(text);
+
   return (
     <div className="note-dialog__note-content">
-      <textarea
-        className="note-dialog__note-content__text"
-        disabled={!editable}
-        onBlur={(e) => onEdit(noteId!, e.target.value ?? "")}
-        onFocus={onFocus}
-        onKeyDown={(e) => {
-          if (e.key === "Enter") {
-            e.preventDefault();
-            (e.target as HTMLTextAreaElement).blur();
-          }
-        }}
-        defaultValue={text}
-      />
+      {isImage ? (
+        <img
+          src={addProtocol(text)}
+          className="note-dialog__note-content--image"
+          alt=""
```

Comment:
You should add an ALT text that for example generally describes that the note displays an image